perun-core/src/main/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntry.java

```diff
@@ -352,7 +353,8 @@ public void assignGroupsToResource(PerunSession perunSession, List<Group> groups
 		Utils.checkPerunSession(perunSession);
 		Utils.notNull(groups, "groups");
 		getResourcesManagerBl().checkResourceExists(perunSession, resource);
-
+		List<PerunBean> beans = new ArrayList<>(groups);
+		beans.add(resource);
```

Comment:
``beans`` is not used anywhere. Its probably part of overall change to authorization, which you don't want to be part of this pull-request.

Reply:
Fixed.